### ISSUE TITLE
.entry-content width for pages match that of posts

### ISSUE THREAD

**Issue by brentlogan** [WordPress/twentysixteen#215]
I think it would simplify adoption of Twenty Sixteen if the page template structure matched that of posts. For the middle width screens, this would mean:

Title aligned with the left
Wide featured image
.entry-content narrower for ease in reading

**Comment by iamtakashi** [comment]
What exactly the range of screen size do you mean by the "middle screen size"?

What about larger screen than the "middle screen size"?

It's worth noting that not everyone uses featured image. I don't want featured image to be a critical layout element.

If you're suggesting design change, it'd be helpful to attach screenshots of how it'd look so that we can see exactly what you're suggesting.

**Comment by karmatosed** [comment]
I think what is being suggested would be to have the content full width, but absolutely agree it would be great to see a mockup of this so or screenshots of an issue. It's hard to make sure we're on same page otherwise @brentlogan.

**Comment by brentlogan** [comment]
Sure. Here is a sample post with a featured image. Notice how the text is indented from the left edge. I realize that the profile pic, title, and other post meta fits in the margin, but the narrower line width makes the text easier to read. (Note: to be able to capture more, I set the browser's zoom to 75%.)

![post](https://cloud.githubusercontent.com/assets/14060692/9983917/c4c63fa0-5fc1-11e5-9b4b-3e27f63ff544.png)

And here is a sample page, as currently designed in Twenty Sixteen.

![page - full width](https://cloud.githubusercontent.com/assets/14060692/9983923/dc774c20-5fc1-11e5-83f5-68b59f9e299c.png)

The text line width is quite long, making it less than optimum for readability. Shortening the line width to match that of the post looks like this.

![page - match post width](https://cloud.githubusercontent.com/assets/14060692/9983932/fff22328-5fc1-11e5-8eca-bd9688dd5010.png)

The narrower width of the text makes it easier to read.

With 5 width breakpoints, it's probably harder for the typical WordPress user to make these changes to page formatting.

**Comment by iamtakashi** [comment]
Thanks for the mockup. I see now what you're suggesting.

I actually have thought about this beginning. But I've never liked it for pages.

With a featured image, it looks "OK-ish" but I don't want a featured image to be an essential element for a layout because not everyone use it, and without a featured image and the edit link which only admins can see, this is how it'd look.

![screen shot 2015-09-21 at 02 31 25](https://cloud.githubusercontent.com/assets/908665/9988407/30f59898-6048-11e5-8d47-cc04ce5815f6.png)

I'm all for better readability, and that's the one of the reason I made the layout for posts. But this layout for pages where we don't have entry meta feels really weird. 

Regarding readability, it's crucial but it's not as much as it is for posts in my opinion. The character per line for the current page layout is around 105 - it is less readable than posts for sure but I don't think it's the end of the world for pages. The wider page column could be beneficial especially on pages with forms, tables, videos and shortcodes too.

I appreciate your suggestion but as I had dismissed already, I don't think this is a great idea.
